source/manual/local-frontend-development.html.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Local frontend development
+parent: "/manual.html"
+layout: manual_layout
+section: Frontend
+type: learn
+owner_slack: "#govuk-2ndline"
+last_reviewed_on: 2019-07-08
+review_in: 6 months
+---
+
+Make the following changes to your setup if you are making changes to frontend apps and need to view those changes locally.
+
+In the instructions below we are running the [frontend] app, but would work for any of the other frontend apps such as [government-frontend].
+
+## Running just a frontend app against live data:
+
+```shell
+$dev cd /var/govuk/frontend
+$dev ./startup.sh --live
+```
+
+## Running a frontend app against changes in `govuk_publishing_components`
+
+Update the Gemfile for the frontend app to point to your local `govuk_publishing_components` gem:
+
+```ruby
+gem 'govuk_publishing_components', path: '../govuk_publishing_components'
+```
+
+Then run your frontend app again pointing to live:
+
+```shell
+$dev ./startup.sh --live
+```
+
+## Running a frontend app against changes in `static`
+
+This is only relevant if the change you are making to the [govuk_publishing_components] gem affects [static]. It's likely to only be things that we have across the whole site, e.g: cookie banner (and survey banner possibly).
+
+Make changes to the Gemfile of the frontend app as per above, then set in `development.rb`:
```

Comment:
It's not clear here what changes you are referring to..? Do you mean the `govuk_publishing_components` change? Can we add a link to the relevant heading?